### DISCUSSION TITLE
Broadcast view_as_consumer

### DISCUSSION THREAD

**Discussion by juliGer** [NilCoalescing/djangochannelsrestframework#141] [FAQ] [answered]
Hello @hishnash I wanted to ask you a question, how do you send a message to all the users that are connected to the channel using view_as_consumer? and I also wanted to know how to add groups, if you have an example of how to do this it would be epic

Thanks greetings!

**Comment by hishnash** [comment, marked as answer]
`view_as_consumer` is not intended for this use case, if you need to communicate with the open connection you will need to use the `ApiConsumer` or `GenericAsyncAPIConsumer` consumer base classes since any communication over the channel will require async methods.
